src/__experimental__/components/checkbox/checkbox.spec.js

```diff
@@ -223,14 +227,14 @@ describe("Checkbox", () => {
       });
     });
 
-    describe("with a left margin (ml prop)", () => {
+    describe("with a left margin", () => {
```

Comment:
I think these tests can be removed because `testStyledSystemMargin` covers all scenarios

Reply:
Actually we can't. It's related to https://github.com/Sage/carbon/pull/4020/files#diff-a6d9af8f0ac0cac088d35a4b74ddfb836ba3c33dc29bb498898f427b3dda90a5R32.